contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -293,27 +328,25 @@ func runEventLoop(
 			)
 
 			switch key.Upper() {
-			case rawterm.KeyH:
+			case rawterm.KeyH: // Helper
 				logger.Info("Gno Dev Helper", "helper", helper)
-			case rawterm.KeyR:
+			case rawterm.KeyA: // Accounts
+				logAccounts(logger.WithGroup(AccountsLogName), kb, dnode)
+			case rawterm.KeyR: // Reload
 				logger.WithGroup(NodeLogName).Info("reloading...")
 				if err = dnode.ReloadAll(ctx); err != nil {
 					logger.WithGroup(NodeLogName).
 						Error("unable to reload node", "err", err)
-
 				}
 
-			case rawterm.KeyCtrlR:
+			case rawterm.KeyCtrlR: // Reset
 				logger.WithGroup(NodeLogName).Info("reseting node state...")
 				if err = dnode.Reset(ctx); err != nil {
 					logger.WithGroup(NodeLogName).
 						Error("unable to reset node state", "err", err)
 				}
 
-			case rawterm.KeyCtrlC:
-				return nil
-			case rawterm.KeyCtrlE:
-				panic("NOOOO")
```

Comment:
💀

Reply:
How did this even go in, in the first place 😄

Reply:
`linting` wasn't working in the first place, but this probably should not have been caught anyway. 🙃